### ISSUE TITLE
MissingMethodException when starting PowerShell Session (EditorServices)

### ISSUE THREAD

**Issue by gabfv** [PowerShell/vscode-powershell#4630]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [X] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

Type: <b>Bug</b>

When starting EditorServices for PowerShell. Here's the error message:

```
PowerShell Extension v2023.6.0
Copyright (c) Microsoft Corporation.

https://aka.ms/vscode-powershell
Type 'help' to get help.

Exception encountered starting EditorServices. Exception logged in D:\a\_work\1\s\src\PowerShellEditorServices.Hosting\Commands\StartEditorServicesCommand.cs on line 236 in EndProcessing:      
System.MissingMethodException: Méthode introuvable : '!0 OmniSharp.Extensions.JsonRpc.JsonRpcOptionsRegistryBase`1.WithServices(System.Action`1<Microsoft.Extensions.DependencyInjection.IServiceCollection>)'.
   à Microsoft.PowerShell.EditorServices.Server.PsesLanguageServer.<StartAsync>b__14_0(LanguageServerOptions options)
   à OmniSharp.Extensions.LanguageServer.Server.LanguageServer.From(Action`1 optionsAction, IServiceProvider outerServiceProvider, CancellationToken cancellationToken)
   à Microsoft.PowerShell.EditorServices.Server.PsesLanguageServer.<StartAsync>d__14.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Microsoft.PowerShell.EditorServices.Hosting.EditorServicesRunner.<CreateEditorServicesAndRunUntilShutdown>d__9.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Microsoft.PowerShell.EditorServices.Commands.StartEditorServicesCommand.EndProcessing()
```

VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:57:04.379Z)
OS version: Windows_NT x64 10.0.19044
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.68GB (6.97GB free)|
|Process Argv|--crash-reporter-id 69b9c32f-c375-4f66-9b8f-cfe3122fa474|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
xml|Dot|2.5.1
prettier-vscode|esb|9.13.0
vscode-diff|fab|1.4.2
copilot-chat|Git|0.2.1
copilot-nightly|Git|1.89.163
todo-tree|Gru|0.0.226
dupchecker|jia|0.1.7
intellij-idea-keybindings|k--|1.5.9
vscode-autohotkey-plus-plus|mar|3.3.0
git-graph|mhu|1.30.0
hexeditor|ms-|1.9.11
powershell|ms-|2023.6.0
vscode-sanddance|msr|4.1.0
indent-rainbow|ode|8.3.1
partial-diff|ryu|1.4.3
vbsvscode|Ser|1.2.1
sort-lines|Tyr|1.10.1
win-ca|uko|3.5.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes516:30244333
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
a2ce3375:30757347
24365598:30736109
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixhidewlkth:30730051
showindicator:30766890
pythongtdpath:30769146
i26e3531:30769768
dh2dc718:30763024

```

</details>

### PowerShell Version

```console
Name                           Value                                                                                   
----                           -----                                                                                   
PSVersion                      5.1.19041.3031                                                                          
PSEdition                      Desktop                                                                                 
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
BuildVersion                   10.0.19041.3031                                                                         
CLRVersion                     4.0.30319.42000                                                                         
WSManStackVersion              3.0                                                                                     
PSRemotingProtocolVersion      2.3                                                                                     
SerializationVersion           1.1.0.1
```

### Visual Studio Code Version

```console
1.79.2
x64
```

### Extension Version

```console
ms-vscode.powershell@2023.6.0
```

### Steps to Reproduce

1. Open VSCode
2. Open any Powershell script (ps1 or psm1 file)

### Visuals

![image](https://github.com/PowerShell/vscode-powershell/assets/24780542/2af6d521-7a28-4159-818d-b205761b3b2e)

### Logs
```
DÉBOGUER : Logging started
DÉBOGUER : Beginning EndProcessing block
COMMENTAIRES : Removing PSReadLine
DÉBOGUER : Creating host configuration
DÉBOGUER : Determining REPL kind
DÉBOGUER : REPL configured as Legacy
DÉBOGUER : Configuring LSP transport
DÉBOGUER : Configuring debug transport
DÉBOGUER : Session file writer created
COMMENTAIRES : Adding AssemblyResolve event handler for dependency loading
COMMENTAIRES : Loading EditorServices
DÉBOGUER : Assembly resolve event fired for Microsoft.PowerShell.EditorServices, Version=3.8.7.0, Culture=neutral, PublicKeyToken=null
DÉBOGUER : Loading Microsoft.PowerShell.EditorServices, Version=3.8.7.0, Culture=neutral, PublicKeyToken=null from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.PowerShell.EditorServices, Version=3.8.7.0, Culture=neutral, PublicKeyToken=null' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.PowerShell.EditorServices.dll'
DÉBOGUER : Loaded 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\netstandard\v4.0_2.0.0.0__cc7b13ffcd2ddd51\netstandard.dll'
DÉBOGUER : Loaded 'System.Runtime.InteropServices.RuntimeInformation, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Runtime.InteropServices.RuntimeInformation\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Runtime.InteropServices.RuntimeInformation.dll'
COMMENTAIRES : PID: 3032
COMMENTAIRES : 
== Build Details ==
- Editor Services version: 3.8.7
- Build origin:            Release
- Build commit:            128b6090474400ad77e9c68902ab703a87c6ae3b
- Build time:              2023-05-31 00:00:00
COMMENTAIRES : 
== Host Startup Configuration Details ==
 - Host name:                 Visual Studio Code Host
 - Host version:              2023.6.0
 - Host profile ID:           Microsoft.VSCode       
 - PowerShell host type:      System.Management.Automation.Internal.Host.InternalHost

 - REPL setting:              LegacyReadLine
 - Session details path:      c:\Users\REDACTEDUSER\AppData\Roaming\Code\User\globalStorage\ms-vscode.powershell\sessions\PSES-VSCode-20712-378576.json
 - Bundled modules path:      c:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules
 - Additional modules:        PowerShellEditorServices.VSCode
 - Feature flags:

 - Log path:
c:\Users\REDACTEDUSER\AppData\Roaming\Code\User\globalStorage\ms-vscode.powershell\logs\1687444589-9fb6b4fd-6ef2-4ce4-8e60-32732c82466f1687444585850\EditorServices.log
 - Minimum log level:         Diagnostic

 - Profile paths:
   + AllUsersAllHosts:       C:\Windows\System32\WindowsPowerShell\v1.0\profile.ps1
   + AllUsersCurrentHost:    C:\Windows\System32\WindowsPowerShell\v1.0\Microsoft.VSCode_profile.ps1
   + CurrentUserAllHosts:    C:\Users\REDACTEDUSER\OneDrive - REDACTEDORG\Documents\WindowsPowerShell\profile.ps1
   + CurrentUserCurrentHost: C:\Users\REDACTEDUSER\OneDrive - REDACTEDORG\Documents\WindowsPowerShell\Microsoft.VSCode_profile.ps1
DÉBOGUER : Loaded 'Microsoft.PowerShell.Commands.Diagnostics, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Commands.Diagnostics\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.PowerShell.Commands.Diagnostics.dll'
DÉBOGUER : Loaded 'System.Configuration.Install, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Configuration.Install\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Configuration.Install.dll'
DÉBOGUER : Loaded 'Microsoft.PowerShell.Commands.Utility, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Commands.Utility\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.PowerShell.Commands.Utility.dll'
DÉBOGUER : Loaded 'Microsoft.PowerShell.Commands.Management, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Commands.Management\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.PowerShell.Commands.Management.dll'
DÉBOGUER : Loaded 'Microsoft.WSMan.Management, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.WSMan.Management\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.WSMan.Management.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.PowerShell.Security.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Assembly resolve event fired for Microsoft.PowerShell.Security.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Loaded 'Microsoft.PowerShell.Security.resources, Version=3.0.0.0, Culture=fr, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Security.resources\v4.0_3.0.0.0_fr_31bf3856ad364e35\Microsoft.PowerShell.Security.resources.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Assembly resolve event fired for Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Loaded 'Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.WSMan.Management.resources\v4.0_3.0.0.0_fr_31bf3856ad364e35\Microsoft.WSMan.Management.resources.dll'
COMMENTAIRES : 
== Console Details ==
 - Console input encoding: Europe de l'Ouest (DOS)
 - Console output encoding: Europe de l'Ouest (DOS)
 - PowerShell output encoding: US-ASCII
COMMENTAIRES : 
== Environment Details ==
 - OS description:  Microsoft Windows 10.0.19044
 - OS architecture: X64
 - Process bitness: 64
COMMENTAIRES : 
== PowerShell Details ==
- PowerShell version: 5.1.19041.3031
- Language mode:      FullLanguage
COMMENTAIRES : Checking that .NET Framework version is at least 4.8
COMMENTAIRES : .NET registry version: 528372
COMMENTAIRES : Updated PSModulePath to: 'C:\Users\REDACTEDUSER\OneDrive - REDACTEDORG\Documents\WindowsPowerShell\Modules;C:\Program 
Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules;c:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules'
DÉBOGUER : Validating configuration
COMMENTAIRES : Loading PowerShell Editor Services
COMMENTAIRES : Starting EditorServices
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Logging.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Logging.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Logging.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Logging.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Logging.Abstractions.dll'
DÉBOGUER : Assembly resolve event fired for Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.dll'
DÉBOGUER : Assembly resolve event fired for Serilog.Sinks.Async, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog.Sinks.Async, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog.Sinks.Async, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.Sinks.Async.dll'
DÉBOGUER : Assembly resolve event fired for Serilog.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.Extensions.Logging.dll'
DÉBOGUER : Assembly resolve event fired for Serilog.Sinks.File, Version=5.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog.Sinks.File, Version=5.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog.Sinks.File, Version=5.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.Sinks.File.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Options, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Options, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Options, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Options.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.LanguageServer, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.LanguageServer, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.LanguageServer, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.LanguageServer.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.JsonRpc, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.JsonRpc, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.JsonRpc, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.JsonRpc.dll'
DÉBOGUER : Assembly resolve event fired for MediatR, Version=8.0.0.0, Culture=neutral, PublicKeyToken=bb9a41a5e8aaa7e2
DÉBOGUER : Loading MediatR, Version=8.0.0.0, Culture=neutral, PublicKeyToken=bb9a41a5e8aaa7e2 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'MediatR, Version=8.0.0.0, Culture=neutral, PublicKeyToken=bb9a41a5e8aaa7e2' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\MediatR.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.LanguageProtocol, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.LanguageProtocol, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.LanguageProtocol, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.LanguageProtocol.dll'
DÉBOGUER : Assembly resolve event fired for Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed
DÉBOGUER : Loading Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Newtonsoft.Json.dll'
DÉBOGUER : Creating/running editor services
DÉBOGUER : Creating startup info object
COMMENTAIRES : Starting server, deregistering host logger and registering shutdown listener
PowerShell Extension v2023.6.0
Copyright (c) Microsoft Corporation.

https://aka.ms/vscode-powershell
Type 'help' to get help.

DÉBOGUER : Logging started
DÉBOGUER : Beginning EndProcessing block
COMMENTAIRES : Removing PSReadLine
DÉBOGUER : Creating host configuration
DÉBOGUER : Determining REPL kind
DÉBOGUER : REPL configured as Legacy
DÉBOGUER : Configuring LSP transport
DÉBOGUER : Configuring debug transport
DÉBOGUER : Session file writer created
COMMENTAIRES : Adding AssemblyResolve event handler for dependency loading
COMMENTAIRES : Loading EditorServices
DÉBOGUER : Assembly resolve event fired for Microsoft.PowerShell.EditorServices, Version=3.8.7.0, Culture=neutral, PublicKeyToken=null
DÉBOGUER : Loading Microsoft.PowerShell.EditorServices, Version=3.8.7.0, Culture=neutral, PublicKeyToken=null from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.PowerShell.EditorServices, Version=3.8.7.0, Culture=neutral, PublicKeyToken=null' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.PowerShell.EditorServices.dll'
DÉBOGUER : Loaded 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\netstandard\v4.0_2.0.0.0__cc7b13ffcd2ddd51\netstandard.dll'
DÉBOGUER : Loaded 'System.Runtime.InteropServices.RuntimeInformation, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Runtime.InteropServices.RuntimeInformation\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Runtime.InteropServices.RuntimeInformation.dll'
COMMENTAIRES : PID: 3032
COMMENTAIRES : 
== Build Details ==
- Editor Services version: 3.8.7
- Build origin:            Release
- Build commit:            128b6090474400ad77e9c68902ab703a87c6ae3b
- Build time:              2023-05-31 00:00:00
COMMENTAIRES : 
== Host Startup Configuration Details ==
 - Host name:                 Visual Studio Code Host
 - Host version:              2023.6.0
 - Host profile ID:           Microsoft.VSCode
 - PowerShell host type:      System.Management.Automation.Internal.Host.InternalHost

 - REPL setting:              LegacyReadLine
 - Session details path:      c:\Users\REDACTEDUSER\AppData\Roaming\Code\User\globalStorage\ms-vscode.powershell\sessions\PSES-VSCode-20712-378576.json
 - Bundled modules path:      c:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules
 - Additional modules:        PowerShellEditorServices.VSCode
 - Feature flags:

 - Log path:
c:\Users\REDACTEDUSER\AppData\Roaming\Code\User\globalStorage\ms-vscode.powershell\logs\1687444589-9fb6b4fd-6ef2-4ce4-8e60-32732c82466f1687444585850\EditorServices.log
 - Minimum log level:         Diagnostic

 - Profile paths:
   + AllUsersAllHosts:       C:\Windows\System32\WindowsPowerShell\v1.0\profile.ps1
   + AllUsersCurrentHost:    C:\Windows\System32\WindowsPowerShell\v1.0\Microsoft.VSCode_profile.ps1
   + CurrentUserAllHosts:    C:\Users\REDACTEDUSER\OneDrive - REDACTEDORG\Documents\WindowsPowerShell\profile.ps1
   + CurrentUserCurrentHost: C:\Users\REDACTEDUSER\OneDrive - REDACTEDORG\Documents\WindowsPowerShell\Microsoft.VSCode_profile.ps1
DÉBOGUER : Loaded 'Microsoft.PowerShell.Commands.Diagnostics, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Commands.Diagnostics\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.PowerShell.Commands.Diagnostics.dll'
DÉBOGUER : Loaded 'System.Configuration.Install, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Configuration.Install\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Configuration.Install.dll'
DÉBOGUER : Loaded 'Microsoft.PowerShell.Commands.Utility, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Commands.Utility\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.PowerShell.Commands.Utility.dll'
DÉBOGUER : Loaded 'Microsoft.PowerShell.Commands.Management, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Commands.Management\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.PowerShell.Commands.Management.dll'
DÉBOGUER : Loaded 'Microsoft.WSMan.Management, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.WSMan.Management\v4.0_3.0.0.0__31bf3856ad364e35\Microsoft.WSMan.Management.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.PowerShell.Security.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Assembly resolve event fired for Microsoft.PowerShell.Security.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Loaded 'Microsoft.PowerShell.Security.resources, Version=3.0.0.0, Culture=fr, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.PowerShell.Security.resources\v4.0_3.0.0.0_fr_31bf3856ad364e35\Microsoft.PowerShell.Security.resources.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Assembly resolve event fired for Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Loaded 'Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.WSMan.Management.resources\v4.0_3.0.0.0_fr_31bf3856ad364e35\Microsoft.WSMan.Management.resources.dll'
COMMENTAIRES : 
== Console Details ==
 - Console input encoding: Europe de l'Ouest (DOS)
 - Console output encoding: Europe de l'Ouest (DOS)
 - PowerShell output encoding: US-ASCII
COMMENTAIRES : 
== Environment Details ==
 - OS description:  Microsoft Windows 10.0.19044
 - OS architecture: X64
 - Process bitness: 64
COMMENTAIRES : 
== PowerShell Details ==
- PowerShell version: 5.1.19041.3031
- Language mode:      FullLanguage
COMMENTAIRES : Checking that .NET Framework version is at least 4.8
COMMENTAIRES : .NET registry version: 528372
COMMENTAIRES : Updated PSModulePath to: 'C:\Users\REDACTEDUSER\OneDrive - REDACTEDORG\Documents\WindowsPowerShell\Modules;C:\Program 
Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules;c:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules'
DÉBOGUER : Validating configuration
COMMENTAIRES : Loading PowerShell Editor Services
COMMENTAIRES : Starting EditorServices
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Logging.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Logging.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Logging.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Logging.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Logging.Abstractions.dll'
DÉBOGUER : Assembly resolve event fired for Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.dll'
DÉBOGUER : Assembly resolve event fired for Serilog.Sinks.Async, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog.Sinks.Async, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog.Sinks.Async, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.Sinks.Async.dll'
DÉBOGUER : Assembly resolve event fired for Serilog.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog.Extensions.Logging, Version=7.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.Extensions.Logging.dll'
DÉBOGUER : Assembly resolve event fired for Serilog.Sinks.File, Version=5.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10
DÉBOGUER : Loading Serilog.Sinks.File, Version=5.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Serilog.Sinks.File, Version=5.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Serilog.Sinks.File.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Options, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Options, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Options, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Options.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.LanguageServer, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.LanguageServer, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.LanguageServer, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.LanguageServer.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.JsonRpc, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.JsonRpc, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.JsonRpc, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.JsonRpc.dll'
DÉBOGUER : Assembly resolve event fired for MediatR, Version=8.0.0.0, Culture=neutral, PublicKeyToken=bb9a41a5e8aaa7e2
DÉBOGUER : Loading MediatR, Version=8.0.0.0, Culture=neutral, PublicKeyToken=bb9a41a5e8aaa7e2 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'MediatR, Version=8.0.0.0, Culture=neutral, PublicKeyToken=bb9a41a5e8aaa7e2' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\MediatR.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.LanguageProtocol, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.LanguageProtocol, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.LanguageProtocol, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.LanguageProtocol.dll'
DÉBOGUER : Assembly resolve event fired for Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed
DÉBOGUER : Loading Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Newtonsoft.Json.dll'
DÉBOGUER : Creating/running editor services
DÉBOGUER : Creating startup info object
COMMENTAIRES : Starting server, deregistering host logger and registering shutdown listener
COMMENTAIRES : Creating LSP transport with endpoint InOut pipe: PSES_kfmmcbdh.qf4
DÉBOGUER : Creating named pipe
DÉBOGUER : Waiting for named pipe connection
DÉBOGUER : Writing session file
DÉBOGUER : Writing session started
DÉBOGUER : Assembly resolve event fired for Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Assembly resolve event fired for Microsoft.WSMan.Management.resources, Version=3.0.0.0, Culture=fr-CA, PublicKeyToken=31bf3856ad364e35
DÉBOGUER : Loaded 'System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Web.Extensions\v4.0_4.0.0.0__31bf3856ad364e35\System.Web.Extensions.dll'
DÉBOGUER : Loaded 'System.Web, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_64\System.Web\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Web.dll'
COMMENTAIRES : Session file written to c:\Users\REDACTEDUSER\AppData\Roaming\Code\User\globalStorage\ms-vscode.powershell\sessions\PSES-VSCode-20712-378576.json with content:
{"status":"started","languageServiceTransport":"NamedPipe","languageServicePipeName":"\\\\.\\pipe\\PSES_kfmmcbdh.qf4","debugServiceTransport":"NamedPipe","debugServicePipeName":"\\\\.\\pipe 
\\PSES_ics0qj3r.kij","powerShellVersion":"5.1.19041.3031"}
DÉBOGUER : Waiting on PSES run/shutdown
DÉBOGUER : Named pipe connected
DÉBOGUER : Creating language server
COMMENTAIRES : Creating debug adapter transport with endpoint InOut pipe: PSES_ics0qj3r.kij
DÉBOGUER : Creating named pipe
DÉBOGUER : Waiting for named pipe connection
DÉBOGUER : Assembly resolve event fired for System.Collections.Immutable, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
DÉBOGUER : Loading System.Collections.Immutable, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'System.Collections.Immutable, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\System.Collections.Immutable.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Configuration, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Configuration, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Configuration, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Configuration.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.Configuration.Abstractions, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.Configuration.Abstractions, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'Microsoft.Extensions.Configuration.Abstractions, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.Configuration.Abstractions.dll'
DÉBOGUER : Loaded 'Microsoft.Extensions.Logging, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.Extensions.Logging\v4.0_6.0.0.0__adb9793829ddae60\Microsoft.Extensions.Logging.dll'
DÉBOGUER : Assembly resolve event fired for System.Reactive, Version=5.0.0.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263
DÉBOGUER : Loading System.Reactive, Version=5.0.0.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'System.Reactive, Version=5.0.0.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\System.Reactive.dll'
DÉBOGUER : Loaded 'Microsoft.Extensions.DependencyInjection.Abstractions, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.Extensions.DependencyInjection.Abstractions\v4.0_6.0.0.0__adb9793829ddae60\Microsoft.Extensions.DependencyInjection.Abstractions.dll'   
DÉBOGUER : Loaded 'Microsoft.Extensions.Options, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.Extensions.Options\v4.0_6.0.0.0__adb9793829ddae60\Microsoft.Extensions.Options.dll'
DÉBOGUER : Loaded 'Microsoft.Extensions.Logging.Abstractions, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\Microsoft.Extensions.Logging.Abstractions\v4.0_6.0.0.0__adb9793829ddae60\Microsoft.Extensions.Logging.Abstractions.dll'
DÉBOGUER : Assembly resolve event fired for Microsoft.Extensions.DependencyInjection.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
DÉBOGUER : Loading Microsoft.Extensions.DependencyInjection.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 from PSES dependency dir into LoadFile context    
DÉBOGUER : Loaded 'Microsoft.Extensions.DependencyInjection.Abstractions, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\Microsoft.Extensions.DependencyInjection.Abstractions.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.DebugAdapter.Server, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.DebugAdapter.Server, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.DebugAdapter.Server, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.DebugAdapter.Server.dll'
DÉBOGUER : Assembly resolve event fired for OmniSharp.Extensions.DebugAdapter, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022
DÉBOGUER : Loading OmniSharp.Extensions.DebugAdapter, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022 from PSES dependency dir into LoadFile context
DÉBOGUER : Loaded 'OmniSharp.Extensions.DebugAdapter, Version=0.19.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022' from 
'C:\Users\REDACTEDUSER\.vscode\extensions\ms-vscode.powershell-2023.6.0\modules\PowerShellEditorServices\bin\Common\OmniSharp.Extensions.DebugAdapter.dll'
DÉBOGUER : Loader disposed
Exception encountered starting EditorServices. Exception logged in D:\a\_work\1\s\src\PowerShellEditorServices.Hosting\Commands\StartEditorServicesCommand.cs on line 236 in EndProcessing:
System.MissingMethodException: Méthode introuvable : '!0 OmniSharp.Extensions.JsonRpc.JsonRpcOptionsRegistryBase`1.WithServices(System.Action`1<Microsoft.Extensions.DependencyInjection.IServiceCollection>)'.
   à Microsoft.PowerShell.EditorServices.Server.PsesLanguageServer.<StartAsync>b__14_0(LanguageServerOptions options)
   à OmniSharp.Extensions.LanguageServer.Server.LanguageServer.From(Action`1 optionsAction, IServiceProvider outerServiceProvider, CancellationToken cancellationToken)
   à Microsoft.PowerShell.EditorServices.Server.PsesLanguageServer.<StartAsync>d__14.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Microsoft.PowerShell.EditorServices.Hosting.EditorServicesRunner.<CreateEditorServicesAndRunUntilShutdown>d__9.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Microsoft.PowerShell.EditorServices.Commands.StartEditorServicesCommand.EndProcessing()

== Press any key to close terminal ==
```

**Comment by SydneyhSmith** [comment]
Thanks @gabfv for opening this issue, sorry that you are hitting this... it looks like an issue with the GAC which happens on Windows PowerShell 5.1, the workaround is to use PowerShell 7...it is a duplicate of https://github.com/PowerShell/vscode-powershell/issues/2538

**Comment by gabfv** [comment]
Thanks for the follow-up! I'll close this issue since this is a duplicate and I'll try pushing for PowerShell 7 in my org (we don't have admin rights for a proper install). I've tried the zip installation method for PS7 and the extension works with it.

**Comment by andyleejordan** [comment]
@gabfv for what it's worth, I believe v2024.5.1-preview of the extension fixes this as we removed the conflicting Serilog dependency entirely, so there should be fewer (though not zero) potential conflicts with the GAC when using PowerShell 5.1.
